External/Utilities/Opacities/Bruenn85/Chimera/abemrgn_weaklib.f90

```diff
@@ -107,8 +107,8 @@ SUBROUTINE abemrgn_weaklib &
         ( n, nez, e_in, rho, t, xh, ah, zh, cmpn, cmpp,  &
           cmpe, absrnc, emisnc )
 
-  absor = absrnp + absrnc
-  emit  = emisnp + emisnc
+  absor = absrnp !+ absrnc
+  emit  = emisnp !+ emisnc
```

Comment:
Are we intending to commit this to `master`?
We still want the Abem table to have the heavies for now, right? @endeve